### PR TITLE
added Django Forbid

### LINKED ISSUE #946: [Add] Django Forbid
### Site URL

https://docs.pysnippet.org/django-forbid/

### Description of the above provided site

The above-provided site is the up-to-date documentation of the tool. It is a new and growing OSS that aims to forbid those who are geolocated in forbidden countries/regions/territories, it also has other features like VPN detection. You can learn more about the project [here](https://github.com/pysnippet/django-forbid).

### Category

Backend

### Sub Category

security

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [ ] I am willing to work on this issue (optional)
- [ ] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by Bhanuteja01**
## Fixes Issue
closes #946 

## Changes proposed
added Django forbid in the backend

### CODE REVIEW

File: database/backend/security.json [hunk 1]
```diff
@@ -1 +1,9 @@
-[]
+[
+  {
+    "name": "Django Forbid",
+    "description": "Django Forbid provides documentation for a growing OSS tool forbidding geolocated individuals in forbidden countries, regions, and territories.",
```
Comment (ArtyomVancyan) [inline database/backend/security.json:4]:
```suggestion
    "description": "Secure your Django app by controlling the access - grant or deny user access based on device and location, including VPN detection.",
```

Comment (ArtyomVancyan) [reply database/backend/security.json:4]: Hey @Bhanuteja01, thanks for adding [Django Forbid](https://github.com/pysnippet/django-forbid). Could you please change the description of the project as it is in its GitHub repository?

File: database/backend/security.json [hunk 2]
```diff
@@ -1 +1,9 @@
-[]
+[
+  {
+    "name": "Django Forbid",
+    "description": "Secure your Django app by controlling the access - grant or deny user access based on device and location, including VPN detection.",
```
Comment (CBID2) [inline database/backend/security.json:4]: [Proofreading]
```suggestion
    "description": "This source helps grant or deny user access to Django apps based on device and location, including VPN detection.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi there @Bhanuteja01. I provided a suggestion on how improve the description! 😀

**Review by CBID2 — APPROVED** [review]
LGTM!

**Review by ujjawaltyagii — APPROVED** [review]
LGTM

**Review by rupali-codes — APPROVED** [review]

**Pull request merged**